udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -49,6 +49,9 @@
 
 {% block toolbar %}
 {{ follow_btn(dataset) }}
+<div class="fr-col-auto fr-ml-3v vuejs">
```

Comment:
According to https://github.com/datagouv/data.gouv.fr/issues/381#issuecomment-2074838647, we should not display this report button if the organization is certified.